src/couch_replicator_scheduler_job.erl

```diff
@@ -332,9 +334,9 @@ handle_cast({db_compacted, DbName},
     {noreply, State#rep_state{target = NewTarget}};
 
 handle_cast(checkpoint, State) ->
-    #rep_state{rep_details = #rep{} = Rep} = State,
-    case couch_replicator_manager:continue(Rep) of
-    {true, _} ->
+    #rep_state{rep_details = #rep{db_name = DbName, doc_id = DocId} = Rep} = State,
+    case couch_replicator_clustering:owner(DbName, DocId) of
```

Comment:
I think I got confused thinking to have a #rep{} we'd have to go through computing the replication_id